### PR TITLE
XW-1719 icons

### PR DESCRIPTION
**Pull request by kamilkoterba**
## Links

* http://wikia-inc.atlassian.net/browse/XW-1719

## Description

Add design system icons to Mercury

## Reviewers

@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: server/app/views/_partials/global-footer/global-footer-svg-logo.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg class="global-footer__{{imageSizeModifier}}{{section}}-logo">
```
Comment (kvas-damian) [inline server/app/views/_partials/global-footer/global-footer-svg-logo.hbs]: Do we need 2 separate partials for changing class name?

File: server/app/views/_partials/global-footer/global-footer.hbs [hunk 2]
```diff
@@ -2,9 +2,7 @@
 	{{#if isInternational}}
 		<h2 class="global-footer__header">
 			{{#with model.international_header.header as |header|}}
-				<img class="global-footer__large-wikia-logo"
-					 src="http://public.diana.wikia-dev.com/design-system-reference-page/bower_components/design-system/assets/{{header.image}}.svg"
-					 alt="{{header.title.key}}">
+				{{> global-footer/global-footer-svg image=header.image section='wikia' imageSizeModifier='large'}}
```
Comment (kvas-damian) [inline server/app/views/_partials/global-footer/global-footer.hbs]: Do we have this partial defined somewhere?

File: server/app/views/_partials/global-footer/global-footer.hbs [hunk 3]
```diff
@@ -2,10 +2,8 @@
 	{{#if isInternational}}
 		<h2 class="global-footer__header">
 			{{#with model.international_header.header as |header|}}
-				<img class="global-footer__large-wikia-logo"
-					 src="http://public.diana.wikia-dev.com/design-system-reference-page/bower_components/design-system/assets/{{header.image}}.svg"
-					 alt="{{header.title.key}}">
-				<span class="global-footer__home-of-fandom">{{header.subtitle.key}}</span>
+				{{> global-footer/global-footer-svg image=header.image section='wikia' imageSizeModifier='large'}}
```
Comment (kvas-damian) [inline server/app/views/_partials/global-footer/global-footer.hbs]: Do we have this partial defined?

File: server/app/views/_partials/global-footer/global-footer.hbs [hunk 4]
```diff
@@ -2,10 +2,8 @@
 	{{#if isInternational}}
 		<h2 class="global-footer__header">
 			{{#with model.international_header.header as |header|}}
-				<img class="global-footer__large-wikia-logo"
-					 src="http://public.diana.wikia-dev.com/design-system-reference-page/bower_components/design-system/assets/{{header.image}}.svg"
-					 alt="{{header.title.key}}">
-				<span class="global-footer__home-of-fandom">{{header.subtitle.key}}</span>
+				{{> svg image=header.image section='wikia' imageSizeModifier='large'}}
```
Comment (kvas-damian) [inline server/app/views/_partials/global-footer/global-footer.hbs]: are `section` and `imageSizeModifier` parameters supported by this partial?

**Comment by kvas-damian** [comment]
LGTM

File: server/app/views/_partials/global-footer/global-footer-image-header.hbs [hunk 5]
```diff
@@ -1,7 +1,9 @@
-{{#if model.href}}
-	<a href="{{model.href}}">
-		<img class="global-footer__{{imageSizeModifier}}{{section}}-logo" src="http://public.diana.wikia-dev.com/design-system-reference-page/bower_components/design-system/assets/{{model.image}}.svg" alt="{{model.title.key}}">
-	</a>
-{{else}}
-	<img class="global-footer__{{imageSizeModifier}}{{section}}-logo" src="http://public.diana.wikia-dev.com/design-system-reference-page/bower_components/design-system/assets/{{model.image}}.svg" alt="{{model.title.key}}">
-{{/if}}
+<h2 class="global-footer__{{section}}-header">
+	{{#if model.href}}
```
Comment (rogatty) [inline server/app/views/_partials/global-footer/global-footer-image-header.hbs]: We should decide based on the `model.type`.

**Comment by rogatty** [comment]
One comment but it's not really a part of this PR. LGTM

File: server/app/views/_partials/global-footer/global-footer-image-header.hbs (hunk 5 above)
Comment (kamilkoterba) [inline server/app/views/_partials/global-footer/global-footer-image-header.hbs]: added to parent PR to TODOs checkboxed list

**Pull request merged**